### ISSUE TITLE
-b -D'|' not applied to -O header line

### ISSUE THREAD

**Issue by svenXY** [harelba/q#132]
Hi,

using something like `q -O -b -D'|'`  together does not seem to beautifully format the header line together with the other lines

``` bash
q -O -b -d'|' "SELECT COUNT(*) AS dup, c1 AS Firstname, c2 AS Lastname, c3 AS Userid, c4 AS Email FROM my.csv GROUP BY lower(Email) HAVING COUNT(*) > 1 ORDER BY dup DESC"
```

yields 

```
dup|Firstname|Lastname  |Userid              |Email               
3|Badaboom|Boing|badaa@example.com|badaa@example.com
```

**Comment by harelba** [comment]
@svenXY sorry for the late reply - True, it's a bug, I've fixed it and pushed into master. Will close this issue when version 1.6.0 is officially out.

**Comment by svenXY** [comment]
Cool. Thanks for fixing it.

**Comment by harelba** [comment]
Fixed in v1.6.3
